src/tests/performance/performance.py

```diff
@@ -0,0 +1,96 @@
+import configparser
+import subprocess
+import datetime
+import json
+import sys
+import os
+import re
+
+from pathlib import Path
+
+# needed to find the execution_info.ini
+def get_latest_output_folder(path):
+    output_path_study = os.path.join(path, "output")
+    files = os.listdir(output_path_study)
+    paths = [os.path.join(output_path_study, basename) for basename in files]
+    return max(paths, key=os.path.getctime)
+
+def search_patern_in_file(file_name, pattern):
+    f = open(file_name)
+    s = f.read()
+    f.close()
+    m = re.search(pattern, s)
+    if m:
+        return m.groups(1)[1]
+    else:
+        print("Pattern not found, aborting")
+        exit(1)
+
+def get_git_revision_short_hash() -> str:
+    return subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('ascii').strip()
```

Comment:
```suggestion
    return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()
```
Let us keep the whole commit-id, to avoid possible collisions.